### ISSUE TITLE
[CAIA Support] from AE Profile | Authenticated Menu

### ISSUE THREAD

**Issue by mtcA6** [department-of-veterans-affairs/va.gov-team#62185]
## About your team

- Team name: Authenticated experience
-	OCTO-DE product owner: Samara Straus 
-	Product manager: Travis Cahill
-	Designer: Pam Drouin, Liz Lantz
-	FE engineer: Adam Whitlock
- Accessibility specialist:
- Content writer: 
-	Product/team Slack channel: Slack channel: #accountexp-authexp; #va-profile
  
### What does your team need support for? Check all that apply.
- [ ] Launching one or more unauthenticated pages, online forms, apps, or tools on VA.gov
- [x] Making revisions to one or more existing VA.gov pages
- [ ] Accessibility review of wireframes and/or prototypes
- [ ] Accessibility issue (general request)
- [ ] Accessibility PDF review request
- [ ] Accessibility user research support ([submit a separate, additional request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name))
- [ ] DST component or pattern 
- [x] Something else 
 
### Give a brief description of what your team needs support with:
[See slack conversation ](https://dsva.slack.com/archives/C01K37HRUAH/p1689692487393159)that kicked this off. 

We're looking to run a lightweight experiment by adding links into the "personalized menu" for [Dependents ](https://staging.va.gov/view-change-dependents/view)and [Letters](https://staging.va.gov/records/download-va-letters/).

- [We saw evidence in Liz's logged-in navigation user research from April that we need to elevate dependents](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/information-architecture/research-and-analytics/2023-generative-logged-in-research/comprehensive-research-summary.md#its-hard-to-get-to-tasks-when-theyre-not-obviously-linked-from-the-homepage-my-va-or-the-mega-menu). People had significant trouble finding dependent information on the site, and many people said they expected to find dependent information in the profile.
- In analytics, variations on "dependents" and "letters" make up the majority of the top 10 search terms in the profile. [These are the stats for the last six months](https://dsva.slack.com/archives/C909ZG2BB/p1689612516606549?thread_ts=1689606177.353799&cid=C909ZG2BB).

We're looking for a lighter weight solution than adding sections for dependents and letters in the profile. We thought we'd try this nav experiment first, see if it moves metrics, and determine next steps based on the outcome.

The goal of this ticket is to understand what we need to keep in mind, and to get input from the content and IA experts. 
 
### Will this new product be released incrementally (for instance 25% of users initially)? 
  - [ ] Yes
  - [x] No (probably no since it's a very minor change)
 
**Note:** If you checked yes, we'll reach out to discuss details about the content in the react widget.
  
### When do you expect to launch your product to 100% of users? 
  
*Please provide an estimated date so our team can create other relevant tickets.*

### When does this work need to be completed?
- Estimated launch date: 9/30/2023
- Estimated staging review date: TBD
- Content, Accessibility, and IA work needed by: 7/28 

### Will this work be going through the Collaboration Cycle?
- [x] Yes 
- [ ] No

### If no ...
- Skip to the _supporting artifacts_ section, below.
 
#### Please provide the link to your Collaboration Cycle ticket: tbd submitting kickoff ticket after this ticket, will provide that ticket momentarily 
- Ticket (number): 
- Collab. Cycle Ticket Opened (date): 
 
#### Please check all of the phases in the Collaboration Cycle you have completed: 
- [ ] Design Intent
- [ ] Midpoint Review
- [ ] Staging
 
##### Design Intent
- [ ] Accessibility
- [x] Content
- [x] IA
- [ ] Translation
- [ ] Already completed, no assistance needed.
- [ ] Unknown at this time.
##### Midpoint
- [ ] Accessibility
- [x] Content
- [x] IA
- [ ] Translation
- [ ] Already completed, no assistance needed.
- [ ] Unknown at this time.
##### Staging
- [ ] Accessibility
- [x] Content
- [x] IA
- [ ] Translation
- [ ] Already completed, no assistance needed.
- [ ] Unknown at this time.
  
## Supporting artifacts
*Please provide supporting artifacts as available.*

- Link to product outline:
- Link to or attach user flows:
- Link to prototype or mockups:
- Link to your research plan: 
- Link to any epics/issues that may be helpful:

#### Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [ ] No, let's work asynchronously and meet if needed

 **Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.
 
### Accessibility Help in Slack
The [#accessibility-help channel](https://dsva.slack.com/archives/C8E985R32) in the DSVA Slack, is also available to you as a resource. Everyone is welcome to ask questions or get help from our a11y (accessibility) specialists. Tag `@Terry Nichols` to get a11y help asap.

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag `@Terry Nichols`.**

- If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
- If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)
 
 - If you are creating an [experimental design,](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) please [also contact the DST. ](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- If you are requesting [a11y support for user research,](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name) please complete this form, next.

**Comment by kristinoletmuskat** [comment]
After talking with this team, we are onboard with this plan. Not only could this be a short-term solution for findability of dependents and letters, but we can also explore whether veterans think to find personalized information in this menu beyond the 3 tools already linked.

Here is a link to [slack thread with screenshot examples from diff tools that leverage personalized menu](https://dsva.slack.com/archives/C909ZG2BB/p1690307602430899).

Here's the [link to the recording](https://drive.google.com/file/d/1q5zG9hSM0tAhv1TE_FbrgVFDKia1sAuq/view?usp=drive_link). 

[Notes](https://docs.google.com/document/d/14sA5NPyXYj_z_TXrwL6N3DLLdqr81pWAtxDX2Q7tLyw/edit?usp=sharing) from the meeting.

**Comment by mtcA6** [comment]
@kristinoletmuskat I think we can close this, what do you think? We're about to have staging review today.

**Comment by coforma-terry** [comment]
Closing for now then @mtcA6 - thank you.
We can always come back if we need to.

CC: @kristinoletmuskat
